config/config.go

```diff
@@ -100,18 +100,21 @@ func init() {
 		TlsConfig.MinVersion = tls.VersionTLS10
 	}
 
-	AuthenticatedUser, err = Authenticator.LoadAuth()
-	if err == nil {
-		C.apiToken = AuthenticatedUser.AuthenticationToken
+	user, tokenGenerator, err := Authenticator.LoadAuth()
```

Comment:
What happens if `err != nil`?

Reply:
user and tokenGenerator will be nil, so no authentication will be loaded and the CLI will run in "unauthenticated" mode. So if a request requires auth, the CLI will ask the user to login